orderfilter/validate_js.go

```diff
@@ -33,13 +39,24 @@ func (s *SchemaValidationResult) Errors() []*SchemaValidationError {
 // ValidateOrderJSON Validates a JSON encoded signed order using the AJV javascript library.
 // This libarary is used to increase the performance of Mesh nodes that run in the browser.
 func (f *Filter) ValidateOrderJSON(orderJSON []byte) (*SchemaValidationResult, error) {
-	jsResult := f.orderValidator.Invoke(string(orderJSON))
-	fatal := jsResult.Get("fatal")
-	if !jsutil.IsNullOrUndefined(fatal) {
-		return nil, errors.New(fatal.String())
+	resultChan := make(chan js.Value, 1)
+	errChan := make(chan error, 1)
+	go func() {
+		time.Sleep(sleepTime)
```

Comment:
`AwaitPromise` and `AwaitPromiseContext` already create a goroutine. Could we potentially add this `time.Sleep` statement there instead? The intention of these functions is to not block the main thread, so if `time.Sleep` is required in order to do that, it feels appropriate to always include it.

Reply:
I don't think so. `AwaitPromise` only creates `go-routines` for `.then` and `.catch`. I'll try it out, but I'm pretty sure that we need to sleep before the promise begins to execute.